projects/components/src/datagrid/interfaces/datagrid-column.interface.ts

```diff
@@ -114,10 +112,12 @@ export interface ContextualButtonConfig<R> {
     buttons: ContextualButton<R>[];
     /**
      * An ordered list of {@link ContextualButton.id}s of buttons that should be in a featured position.
+     * Disabled buttons will not display and only the {@link featuredCount} worth of buttons will be displayed at most.
      */
     featured: string[];
     /**
      * How many buttons should display on the featured section.
+     * Allows the user to specific more {@link featured} buttons than necessary and for some to act as fallbacks.
```

Comment:
Sorry, I don't understand the comment. If I remember right... you said this property is to select first 'n' non-disabled buttons from the featured list. And you said that's necessary to save the screen's real state.